mysql-8.0/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM    gitpod/workspace-full
+
+
+##
+## -- Set up the environment for a Docker build
+##    -----------------------------------------
+USER    root
+RUN     apt-get update 
+RUN     apt-get install debconf-utils -y -q

```

Comment:
Small note in passing, every `RUN` instruction creates a new Docker layer that can be cached.

With `apt-get update` and `apt-get intall`, it's important to have both in the same `RUN` command. Because otherwise what can happen is that the result of `apt-get update` gets cached in an old Docker layer, and if you modify the `apt-get install` instruction (e.g. by adding a package), it might try to use an out-dated list.

What we generally do is:

```Dockerfile
USER gitpod
RUN sudo apt-get update \
 && sudo apt-get install -y \
    package \
 && sudo rm -rf /var/lib/apt/lists/*
```

That way, in a single layer, you download an up-to-date apt list, then install what you need, and then remove the unnecessary apt list again before saving the resulting file system into a Docker layer (thus saving valuable disk space in your image).